Lesson24-signup-form/assets/assets/ezgi/styles.css

```diff
@@ -0,0 +1,143 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: Arial, sans-serif;
+    min-width:320px ;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #3c3952;
+  }
+   #confirm{
+    display: none;
+    background-color: #fff;
+  } 
+
+  .confirm-active{
+border: 1px solid black;
+border-radius: 20px;
+font-weight:lighter;
+    width: 400px;
+    text-align: left;
+    padding: 20px;
+  } 
+
+
+  
+  #container{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 800px;
+    background-color: #fff;
+    padding: 15px;
+    border-radius: 8px;
+  }
+  
+  
+  .form-container {
+    width: 350px;
```

Comment:
Instead of absolute width, use relative value here for more flexibility, also, add a bit of padding on the left:
```
width: 50%;
padding-left: 20px;
```